gradle-scripts/docker.gradle

```diff
@@ -1,6 +1,6 @@
 docker {
     javaApplication {
-        baseImage = 'java:openjdk-8-jre-alpine'
+        baseImage = 'adoptopenjdk/openjdk11:jre-11.0.9.1_1-alpine'
```

Comment:
Curious why not the official one as also why fixed version?
https://hub.docker.com/layers/openjdk/library/openjdk/11.0-jre-slim/images/sha256-e506a736cd0c93ef98666e6e88e4c82fb33f9cfd11df78f70971aaeafd00956f?context=explore

Reply:
It is because openjdk doesn't provide alpine version for java-11. Regarding to image size, 11-jre-slim is still big when comparing with alpine version JRE provided by adoptopenjdk.

Fixed version lacks flexibility here. I found another jre-11 alpine version provided by adoptopenjdk
https://hub.docker.com/layers/adoptopenjdk/openjdk11/alpine-jre/images/sha256-a775c6f9a678eb3db93b65d37c29d4149c93df7df48d76259525ce37769714f9?context=explore

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/254/commits/6cee36aa6defb46b73b4e7ec04864b6777881d41